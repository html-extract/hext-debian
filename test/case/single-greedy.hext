<+h1 class:x />
