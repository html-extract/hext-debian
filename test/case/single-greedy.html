<div class="result-set">
  <h1 class="1">Heading</h1>
  <h1 class="2">Heading</h1>
  <h1 class="3">Heading</h1>
  <h1 class="4">Heading</h1>
</div>
<div class="result-set">
  <h1 class="5">Heading</h1>
</div>
<!-- result-set -->
<h1 class="6">Heading</h1>
<h1 class="7">Heading</h1>
<h1 class="8">Heading</h1>
<h1 class="9">Heading</h1>
<h1 class="10">Heading</h1>
