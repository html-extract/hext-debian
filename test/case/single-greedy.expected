{"x":["6","7","8","9","10"]}
{"x":["1","2","3","4"]}
{"x":"5"}
